spec/consensus/proposer-based-timestamp/tla/TendermintPBT_002_draft.tla

```diff
@@ -255,30 +283,37 @@ BenignRoundsInMessages(msgfun) ==
 \* The initial states of the protocol. Some faults can be in the system already.
 Init ==
     /\ round = [p \in Corr |-> 0]
-    /\ \/ /\ ~ClockDrift
-          /\ localClock \in [Corr -> 0..Accuracy]
-       \/ /\ ClockDrift
-          /\ localClock = [p \in Corr |-> 0]
+    /\ localClock \in [Corr -> MinTimestamp..(MinTimestamp + Precision)]
     /\ realTime = 0
     /\ step = [p \in Corr |-> "PROPOSE"]
     /\ decision = [p \in Corr |-> NilDecision]
     /\ lockedValue = [p \in Corr |-> NilValue]
     /\ lockedRound = [p \in Corr |-> NilRound]
-    /\ validValue = [p \in Corr |-> NilValue]
+    /\ validValue = [p \in Corr |-> NilProposal]
     /\ validRound = [p \in Corr |-> NilRound]
     /\ msgsPropose \in [Rounds -> SUBSET AllFaultyProposals]
     /\ msgsPrevote \in [Rounds -> SUBSET AllFaultyPrevotes]
     /\ msgsPrecommit \in [Rounds -> SUBSET AllFaultyPrecommits]
-    /\ receivedTimelyProposal = EmptyRcvProp
-    /\ inspectedProposal = [r \in Rounds |-> EmptyProcSet]
+    /\ receivedTimelyProposal = [p \in Corr |-> {}]
+    /\ inspectedProposal = [r \in Rounds, p \in Corr |-> NilTimestamp]
     /\ BenignRoundsInMessages(msgsPropose)
     /\ BenignRoundsInMessages(msgsPrevote)
     /\ BenignRoundsInMessages(msgsPrecommit)
-    /\ evidence = EmptyMsgSet
+    /\ evidence = {}
     /\ action' = "Init"
```

Comment:
TLC is not happy about this. In its usual cryptic way: "In evaluation, the identifier action is either undefined or not an operator." This fixes it:
```suggestion
    /\ action = "Init"
```
But then we run into deadlock, with `AdvanceRealTime`.

Reply:
Huh, I'm surprised this one slipped by. Apparently this has been here since v1: https://github.com/tendermint/tendermint/blob/a22942504ccda05e161283d9eb0e5f10fce6d3f1/spec/consensus/proposer-based-timestamp/tla/TendermintPBT_001_draft.tla#L180
It definitely shouldn't be primed here.

Reply:
Apalache has some tolerance for `Init`, where it will treat primes and non-primes inside exactly `Init` as the same, but I suppose TLC would complain.

Reply:
> Huh, I'm surprised this one slipped by. Apparently this has been here since v1:

Yes, it was in v1 as well, i remember fixing it there in my local experiments.